scripts/setup_gaia.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
```

Comment:
To have agents self-contained, let's move these scripts to the agent's directory.